exporters/prometheus/exporter.go

```diff
@@ -92,6 +92,7 @@ type collector struct {
 	scopeInfosInvalid map[instrumentation.Scope]struct{}
 	metricFamilies    map[string]*dto.MetricFamily
 	resourceKeyVals   keyVals
+	metricsPool       sync.Pool
```

Comment:
Why not have the poll be a file level var? Wouldn't that be a better amortization?